python/segyio/segy.py

```diff
@@ -27,14 +27,15 @@ def __init__(
         self,
         fd,
         datasource_descriptor,
-        iline=189,
-        xline=193,
         endian='big'
     ):
 
         self._datasource_descriptor = datasource_descriptor
-        self._il = iline
-        self._xl = xline
+
+        self.traceheader_layouts = fd.traceheader_layouts()
```

Comment:
Maybe _this_ should be private though.
It is likely not the final way we want users to access the data, so maybe we should keep it kind-of-hidden.